src/models/User.ts

```diff
@@ -0,0 +1,6 @@
+export default interface User {
```

Comment:
Name interfaces like that: `IUser`.

Reply:
convention says that i shouldn't use I in interface naming.